pom.xml

```diff
@@ -30,6 +30,11 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
             <version>${version.spring.boot}</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-mongodb</artifactId>
```

Comment:
Does the library tries connect to DB on application start? What happens when DB is down?

Reply:
Yes it does try to connect to localhost 27017, I have updated the readme to reflect this